### PR TITLE
 * add log file and test graph

### PR DESCRIPTION
**Pull request by d-sot**
test results log file and graph.

### CODE REVIEW

File: nwbzarr/nwbzarr.py [hunk 1]
```diff
@@ -102,43 +73,51 @@ def __init__(self, nwbfile: str = None, nwbgroup: str = None, nwbfile_mode: str
...
         if isinstance(self.nwbfile, str):
-            fsspec_file = fsspec.open(self.nwbfile, mode='rb')
-            chunk_store = FileChunkStore(self.store, chunk_source=fsspec_file.open())
+            self.chunkstore_file = fsspec.open(self.nwbfile, mode='rb')
+            self.chunk_store = FileChunkStore(self.store, chunk_source=self.chunkstore_file.open())
         else:
-            chunk_store = FileChunkStore(self.store, chunk_source=self.nwbfile)
-        if LRU is True and not isinstance(chunk_store, zarr.LRUStoreCache):
-            chunk_store = zarr.LRUStoreCache(chunk_store, max_size=self.LRU_max_size)
+            self.chunk_store = FileChunkStore(self.store, chunk_source=self.nwbfile)
```
Comment (bendichter) [inline nwbzarr/nwbzarr.py]: If `nwbfile` can be of type `File`, indicate that in the docstring

File: nwbzarr/tests/logs/logfile-001.csv [hunk 2]
```diff
@@ -0,0 +1,219 @@
+Type,name,Zarr Read Time,h5py: Get Object,h5py: Read Data
```
Comment (bendichter) [inline nwbzarr/tests/logs/logfile-001.csv]: don't check data into GitHub

**Review by bendichter — CHANGES_REQUESTED** [review]
Please rename the class. None of this in NWB-specific. 

Also, please include a README indicating how to run the necessary functions and in what order

File: nwbzarr/nwbzarr.py (hunk 1 above)
Comment (d-sot) [reply nwbzarr/nwbzarr.py]: Class is renamed in 9f277a8. Docstring is updated as well.

File: nwbzarr/tests/logs/logfile-001.csv (hunk 2 above)
Comment (d-sot) [reply nwbzarr/tests/logs/logfile-001.csv]: data is removed.

File: nwbzarr/nwbzarr.py (hunk 1 above)
Comment (d-sot) [reply nwbzarr/nwbzarr.py]: Added an updated README in PR #2

**Pull request merged**